hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/testutils/HiveTestUtil.java

```diff
@@ -82,35 +80,37 @@
 @SuppressWarnings("SameParameterValue")
 public class HiveTestUtil {
 
+  public static HiveSyncConfig hiveSyncConfig;
+  public static FileSystem fileSystem;
   private static ZooKeeperServer zkServer;
   private static HiveServer2 hiveServer;
   public static HiveTestService hiveTestService;
   private static ZookeeperTestService zkService;
   private static Configuration configuration;
-  public static HiveSyncConfig hiveSyncConfig;
   private static DateTimeFormatter dtfOut;
-  public static FileSystem fileSystem;
   private static Set<String> createdTablesSet = new HashSet<>();
 
-  public static void setUp() throws IOException, InterruptedException {
-    configuration = new Configuration();
+  private static TestCluster cluster;
+
+  public static void setUp() throws Exception {
+    if (cluster == null) {
+      cluster = new TestCluster();
+      cluster.setup();
+      configuration = cluster.getConf();
+      fileSystem = cluster.dfsCluster.getFileSystem();
+    }
+    //configuration = new Configuration();
     if (zkServer == null) {
       zkService = new ZookeeperTestService(configuration);
       zkServer = zkService.start();
     }
-    if (hiveServer == null) {
-      hiveTestService = new HiveTestService(configuration);
-      hiveServer = hiveTestService.start();
-    }
-    fileSystem = FileSystem.get(configuration);
-
     hiveSyncConfig = new HiveSyncConfig();
-    hiveSyncConfig.jdbcUrl = hiveTestService.getJdbcHive2Url();
-    hiveSyncConfig.hiveUser = "";
+    hiveSyncConfig.hiveUser = System.getProperty("user.name");
+    hiveSyncConfig.jdbcUrl = cluster.getHiveJdBcUrl();
     hiveSyncConfig.hivePass = "";
     hiveSyncConfig.databaseName = "testdb";
     hiveSyncConfig.tableName = "test1";
-    hiveSyncConfig.basePath = Files.createTempDirectory("hivesynctest" + Instant.now().toEpochMilli()).toUri().toString();
+    hiveSyncConfig.basePath = "/tmp/hdfs/TestHiveSyncTool/";
```

Comment:
Can we use Files.createTempDirectory("TestHiveSyncTool") instead of hard-coding ?

Reply:
Done